mozilla-release/browser/config/cliqz.mozconfig

```diff
@@ -7,7 +7,7 @@ ac_add_options --enable-signmar
 
 ac_add_options --with-branding=browser/branding/cliqz
 ac_add_options --enable-official-branding
-ac_add_options --with-app-name=CLIQZ
+ac_add_options --with-app-name=cliqz  # name for binaries
 
-export MOZ_APP_PROFILE=CLIQZ
+export MOZ_APP_PROFILE=Cliqz
```

Comment:
this probably should be kept CLIQZ all capitals as macos and linux are case sensitive on filesystem names